backend/src/app/rpc/mutations/management.clj

```diff
@@ -84,10 +84,24 @@
                   (d/without-nils)
                   (blob/encode))))))
 
+(def sql:retrieve-used-libraries
+  "select flr.*
+     from file_library_rel as flr
+    inner join file as l on (flr.library_file_id = l.id)
+    where flr.file_id = ?
+      and l.deleted_at is null")
+
+(def sql:retrieve-used-media-objects
+  "select fmo.*
+     from file_media_object as fmo
+    inner join storage_object as o on (fmo.media_id = o.id)
+    where fmo.file_id = ?
+      and (o.deleted_at is null or o.deleted_at > now())")
```

Comment:
I think this should be `o.deleted_at < now()`.

Reply:
finally i have removed the condition